server/service/src/apis/login_v4.rs

```diff
@@ -148,6 +149,11 @@ impl LoginApiV4 {
             return Err(LoginV4Error::Unauthorised);
         }
 
+        // handle account blocked error (i.e. too many failed login attempts)
+        if reqwest::StatusCode::FORBIDDEN == response.status() {
+            return Err(LoginV4Error::AccountBlocked);
+        }
```

Comment:
Right. Have updated with [421963018](https://github.com/msupply-foundation/open-msupply/commit/421963018fafd1f57c50b638577f6361a3331f50#diff-4319a2565ecb0c07ec3c75ecb7a0b2cd3db161a99fdaa51bb38419bae8ab7e19)

Reply:
Nit: while working it is probably nicer to parse it into a matching struct? e.g.

```Rust
#[derive(Deserialize)]
pub struct LoginResponseErrorV4 {
  status: String,
  timeoutRemaining: Option<u64>,
}
```
and then do something like

```Rust
if Ok(error_body) = serde_json::from_value::<LoginResponseErrorV4>(body) {
    ...
};
```
This would also avoid the `.as_u64().unwrap()`

Reply:
d4921fa24